docs/src/app/components/master.jsx

```diff
@@ -1,27 +1,27 @@
 import React from 'react';
 import AppLeftNav from './app-left-nav';
 import FullWidthSection from './full-width-section';
-import {AppBar,
-      AppCanvas,
-      IconButton,
-      EnhancedButton,
-      Mixins,
-      Styles,
-      Tab,
-      Tabs,
-      Paper} from 'material-ui';
-
-const {StylePropable} = Mixins;
-const {Colors, Spacing, Typography} = Styles;
-const ThemeManager = Styles.ThemeManager;
-const DefaultRawTheme = Styles.lightBaseTheme;
+import AppBar from 'material-ui/lib/app-bar';
+import AppCanvas from 'material-ui/lib/app-canvas';
+import IconButton from 'material-ui/lib/icon-button';
+import EnhancedButton from 'material-ui/lib/enhanced-button';
+import Paper from 'material-ui/lib/paper';
+import Tabs from 'material-ui/lib/tabs/tabs';
+import Tab from 'material-ui/lib/tabs/tab';
+import {StylePropable} from 'material-ui/lib/mixins';
 
+import {
+  Colors,
+  Spacing,
+  Typography,
+  ThemeManager,
```

Comment:
you just use the `getMuiTheme` directly. ThemeManager sould be deprecated in the future. since it now makes no sense.